addons/pos_restaurant/static/tests/tours/TicketScreen.tour.js

```diff
@@ -65,3 +65,21 @@ registry.category("web_tour.tours").add("PosResTicketScreenTour", {
             ProductScreen.orderIsEmpty(),
         ].flat(),
 });
+
+registry.category("web_tour.tours").add("OrderNumberConflictTour", {
+    test: true,
+    steps: () =>
+        [
+            FloorScreen.clickTable("3"),
+            ProductScreen.isShown(),
+            ProductScreen.addOrderline("Coca-Cola", "1", "3"),
+            FloorScreen.backToFloor(),
+            Chrome.clickMenuButton(),
+            Chrome.clickTicketButton(),
+            TicketScreen.nthColumnContains(6, 2, "Order"),
+            TicketScreen.nthColumnContains(6, 3, "01"),
+            TicketScreen.nthColumnContains(7, 2, "Self-Order"),
+            TicketScreen.nthColumnContains(7, 3, "S"),
+            TicketScreen.nthColumnContains(7, 3, "01")
+        ].flat(),
+});
```

Comment:
This tour is linked to the self order module so you should put it in the pos_self_order module

Reply:
When trying to add it in pos_self_order I need to import the helper methods and this triggers an error in importing those files. If it is mandatory to have it in pos_self_order I will do it without the helpers methods and will write the whole steps